images/macos/tests/Common.Tests.ps1

```diff
@@ -48,33 +48,136 @@ Describe "Audio device" -Skip:($os.IsHighSierra -or $os.IsBigSur) {
     }
 }
 
-Describe "Common utilities" {
-    It "Homebrew" {
-        "brew --version" | Should -ReturnZeroExitCode
+Describe "Brew common utils" {

```

Comment:
I suggest avoid spaces in describe names. Since we invoke it from bash, it could cause issue.